onedrive_service/src/onedrive_service/util.py

```diff
@@ -0,0 +1,181 @@
+""" Provides service to interact with remote API """
+# pylint: disable=too-many-arguments
+import onedrivesdk
+
+
+class OneDriveUtil(object):
```

Comment:
I would add exception handling so that the user of the class would know if `get_drive()` is called for a non-existent drive, it'll throw an exception of known type.